<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>35</y>
     <width>351</width>
     <height>58</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>造字工房怪魅体（非商用）</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PowerChat 注册</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>125</y>
     <width>300</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>用户名和密码</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_userName">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEdit_userName">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_password">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>305</y>
     <width>300</width>
     <height>109</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>联系方式绑定</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_email">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>邮箱</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_email">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_phone">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>手机</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_phone">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>425</y>
     <width>407</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>同意用户协议请心中默念“AMD YES!”</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_reg">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>80</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>460</y>
     <width>80</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_userName</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>lineEdit_email</tabstop>
  <tabstop>lineEdit_phone</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>pushButton_reg</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
